Flaking Test: consistent_replicate_s3

### Which jobs are flaking?

cdc_ghpr_integration_test

### Which test(s) are flaking?

consistent_replicate_s3

### Jenkins logs or GitHub Actions link

```log
[2022-07-04T07:23:39.702Z] table consistent_replicate_s3.check1 not exists for 41-th check, retry later

[2022-07-04T07:23:42.310Z] table consistent_replicate_s3.check1 not exists for 42-th check, retry later

[2022-07-04T07:23:44.258Z] table consistent_replicate_s3.check1 not exists for 43-th check, retry later

[2022-07-04T07:23:46.186Z] table consistent_replicate_s3.check1 not exists for 44-th check, retry later

[2022-07-04T07:23:48.351Z] table consistent_replicate_s3.check1 not exists for 45-th check, retry later

[2022-07-04T07:23:50.307Z] table consistent_replicate_s3.check1 not exists for 46-th check, retry later

[2022-07-04T07:23:52.340Z] table consistent_replicate_s3.check1 not exists for 47-th check, retry later

[2022-07-04T07:23:54.311Z] table consistent_replicate_s3.check1 not exists for 48-th check, retry later

[2022-07-04T07:23:56.327Z] table consistent_replicate_s3.check1 not exists for 49-th check, retry later

[2022-07-04T07:23:58.316Z] table consistent_replicate_s3.check1 not exists for 50-th check, retry later

[2022-07-04T07:24:00.294Z] table consistent_replicate_s3.check1 not exists for 51-th check, retry later

[2022-07-04T07:24:02.279Z] table consistent_replicate_s3.check1 not exists for 52-th check, retry later

[2022-07-04T07:24:04.274Z] table consistent_replicate_s3.check1 not exists for 53-th check, retry later

[2022-07-04T07:24:06.300Z] table consistent_replicate_s3.check1 not exists for 54-th check, retry later

[2022-07-04T07:24:08.341Z] table consistent_replicate_s3.check1 not exists for 55-th check, retry later

[2022-07-04T07:24:10.411Z] table consistent_replicate_s3.check1 not exists for 56-th check, retry later

[2022-07-04T07:24:13.045Z] table consistent_replicate_s3.check1 not exists for 57-th check, retry later

[2022-07-04T07:24:14.549Z] table consistent_replicate_s3.check1 not exists for 58-th check, retry later

[2022-07-04T07:24:16.686Z] table consistent_replicate_s3.check1 not exists for 59-th check, retry later

[2022-07-04T07:24:18.718Z] table consistent_replicate_s3.check1 not exists for 60-th check, retry later

[2022-07-04T07:24:20.820Z] table consistent_replicate_s3.check1 not exists at last check

[2022-07-04T07:24:20.820Z] Exiting on signal: INTERRUPT

[2022-07-04T07:24:43.130Z] make: *** [integration_test_mysql] Error 1
```

### Anything else we need to know

- Does this test exist for other branches as well?

- Has there been a high frequency of failure lately?

--- a/tests/integration_tests/consistent_replicate_gbk/run.sh
+++ b/tests/integration_tests/consistent_replicate_gbk/run.sh
@@ -78,7 +78,7 @@ function run() {
 	run_sql "INSERT INTO consistent_replicate_gbk.GBKTABLE VALUES (1, '测试', '中国', '上海', '你好,世界', 0xC4E3BAC3CAC0BDE7);" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
 	run_sql "CREATE table consistent_replicate_gbk.check1(id int primary key);" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
 	check_table_exists "consistent_replicate_gbk.GBKTABLE" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
-	check_table_exists "consistent_replicate_gbk.check1" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+	check_table_exists "consistent_replicate_gbk.check1" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} 120
 	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
 
 	# Inject the failpoint to prevent sink execution, but the global resolved can be moved forward.

--- a/tests/integration_tests/consistent_replicate_nfs/run.sh
+++ b/tests/integration_tests/consistent_replicate_nfs/run.sh
@@ -8,6 +8,11 @@ WORK_DIR=$OUT_DIR/$TEST_NAME
 CDC_BINARY=cdc.test
 SINK_TYPE=$1
 
+stop() {
+	echo $(mysql -h${DOWN_TIDB_HOST} -P${DOWN_TIDB_PORT} -uroot -e "select count(*) from consistent_replicate_s3.USERTABLE;")
+	stop_tidb_cluster
+}
+
 # check resolved ts has been persisted in redo log meta
 function check_resolved_ts() {
 	changefeedid=$1
@@ -44,7 +49,7 @@ function run() {
 	go-ycsb load mysql -P $CUR/conf/workload -p mysql.host=${UP_TIDB_HOST} -p mysql.port=${UP_TIDB_PORT} -p mysql.user=root -p mysql.db=consistent_replicate_nfs
 	run_sql "CREATE table consistent_replicate_nfs.check1(id int primary key);" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
 	check_table_exists "consistent_replicate_nfs.USERTABLE" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
-	check_table_exists "consistent_replicate_nfs.check1" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+	check_table_exists "consistent_replicate_nfs.check1" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} 120
 	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
 
 	# Inject the failpoint to prevent sink execution, but the global resolved can be moved forward.
@@ -68,7 +73,7 @@ function run() {
 	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
 }
 
-trap stop_tidb_cluster EXIT
+trap stop EXIT
 run $*
 check_logs $WORK_DIR
 echo "[$(date)] <<<<<< run test case $TEST_NAME success! >>>>>>"

--- a/tests/integration_tests/consistent_replicate_nfs/run.sh
+++ b/tests/integration_tests/consistent_replicate_nfs/run.sh
@@ -9,6 +9,7 @@ CDC_BINARY=cdc.test
 SINK_TYPE=$1
 
 stop() {
+	# to distinguish whether the test failed in the DML synchronization phase or the DDL synchronization phase
 	echo $(mysql -h${DOWN_TIDB_HOST} -P${DOWN_TIDB_PORT} -uroot -e "select count(*) from consistent_replicate_s3.USERTABLE;")
 	stop_tidb_cluster
 }

--- a/tests/integration_tests/consistent_replicate_s3/run.sh
+++ b/tests/integration_tests/consistent_replicate_s3/run.sh
@@ -35,6 +35,7 @@ stop_minio() {
 }
 
 stop() {
+	# to distinguish whether the test failed in the DML synchronization phase or the DDL synchronization phase
 	echo $(mysql -h${DOWN_TIDB_HOST} -P${DOWN_TIDB_PORT} -uroot -e "select count(*) from consistent_replicate_s3.USERTABLE;")
 	stop_minio
 	stop_tidb_cluster

--- a/tests/integration_tests/consistent_replicate_s3/run.sh
+++ b/tests/integration_tests/consistent_replicate_s3/run.sh
@@ -35,6 +35,7 @@ stop_minio() {
 }
 
 stop() {
+	echo $(mysql -h${DOWN_TIDB_HOST} -P${DOWN_TIDB_PORT} -uroot -e "select count(*) from consistent_replicate_s3.USERTABLE;")
 	stop_minio
 	stop_tidb_cluster
 }
@@ -77,7 +78,7 @@ function run() {
 	go-ycsb load mysql -P $CUR/conf/workload -p mysql.host=${UP_TIDB_HOST} -p mysql.port=${UP_TIDB_PORT} -p mysql.user=root -p mysql.db=consistent_replicate_s3
 	run_sql "CREATE table consistent_replicate_s3.check1(id int primary key);" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
 	check_table_exists "consistent_replicate_s3.USERTABLE" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
-	check_table_exists "consistent_replicate_s3.check1" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+	check_table_exists "consistent_replicate_s3.check1" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} 120
 	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
 
 	# Inject the failpoint to prevent sink execution, but the global resolved can be moved forward.

--- a/tests/integration_tests/consistent_replicate_schedulerv3/run.sh
+++ b/tests/integration_tests/consistent_replicate_schedulerv3/run.sh
@@ -9,6 +9,7 @@ CDC_BINARY=cdc.test
 SINK_TYPE=$1
 
 stop() {
+	# to distinguish whether the test failed in the DML synchronization phase or the DDL synchronization phase
 	echo $(mysql -h${DOWN_TIDB_HOST} -P${DOWN_TIDB_PORT} -uroot -e "select count(*) from consistent_replicate_s3.USERTABLE;")
 	stop_tidb_cluster
 }

--- a/tests/integration_tests/consistent_replicate_schedulerv3/run.sh
+++ b/tests/integration_tests/consistent_replicate_schedulerv3/run.sh
@@ -8,6 +8,11 @@ WORK_DIR=$OUT_DIR/$TEST_NAME
 CDC_BINARY=cdc.test
 SINK_TYPE=$1
 
+stop() {
+	echo $(mysql -h${DOWN_TIDB_HOST} -P${DOWN_TIDB_PORT} -uroot -e "select count(*) from consistent_replicate_s3.USERTABLE;")
+	stop_tidb_cluster
+}
+
 # check resolved ts has been persisted in redo log meta
 function check_resolved_ts() {
 	changefeedid=$1
@@ -68,7 +73,7 @@ function run() {
 	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
 }
 
-trap stop_tidb_cluster EXIT
+trap stop EXIT
 run $*
 check_logs $WORK_DIR
 echo "[$(date)] <<<<<< run test case $TEST_NAME success! >>>>>>"

